frontend/src/static/css/v_authorship.scss

```diff
@@ -20,13 +20,25 @@
 
   .title {
     .contribution {
-      color: mui-color('red');
-
       .fileType {
         align-items: center;
         display: flex;
         margin-left: 3rem;
         margin-top: 1.0rem;
+
+        .checkboxes {
```

Comment:
would it be possible for it to utilize the style from summary given that they have the same properties?

Reply:
can. As this is a common style I shift them to panel.scss